interface/modules/custom_modules/oe-module-dorn/public/results.php

```diff
@@ -0,0 +1,155 @@
+                                        </div>
+                                    </div>                                        
+                                </div> 
+                            <div class="row">
+                                <div class="col-md-3">
+                                    <button type="submit" name="SubmitButton" class="btn btn-primary"><?php echo xlt("Submit") ?></button>
+                                </div>
+                            </div>           
+                        </form>                
+                    </div>
+                </div>  
+                <div class="row">
+                    <div class="col">
+                     <?php
+                        if (empty($datas)) {
+                            echo xlt("No results found");
+                        } else { ?>
+                        <table class="table">
+                            <thead>
+                                <tr>
+                                    <th scope="col"><?php echo xlt("AccountNumber") ?></th>
+                                    <th scope="col"><?php echo xlt("Status") ?></th>
+                                    <th scope="col"><?php echo xlt("Has Abnormal Flags") ?></th>
+                                    <th scope="col"><?php echo xlt("Lab Name") ?></th>
+                                    <th scope="col"><?php echo xlt("Actions") ?></th>
+                                </tr>
+                            </thead>
+                            <tbody>
+                                <?php
+                                foreach ($datas as $data) {
+                                    ?>
+                                <tr>
+                                    <td scope="row"><?php echo text($data->accountNumber); ?></td>
+                                    <td scope="row"><?php echo text($data->status); ?></td>
+                                    <td scope="row"><?php echo text($data->hasAbnormalFlags); ?></td>
+                                    <td scope="row"><?php echo text($data->labName); ?></td>
+                                    <td scope="row">
+                                        <button type="button" class="btn btn-primary" onclick="getResults('<?php echo text($data->resultGuid); ?>')"><?php echo xlt('Retrieve Results');?></button>
+                                   
+                                    <?php if (!$data->isPending) { ?>                                        
+                                        <button type="button" class="btn btn-primary" onclick="ackResults('<?php echo text($data->resultGuid); ?>','false')"><?php echo xlt('Accept Results');?></button>
+                                        <button type="button" class="btn btn-primary" onclick="ackResults('<?php echo text($data->resultGuid); ?>','true')"><?php echo xlt('Reject Results');?></button>
```

Comment:
in 3 lines above in the onclick use attr_js instead of text and remove the single quotes around them